nixos/modules/services/networking/wg-access-server.nix

```diff
@@ -104,7 +104,7 @@ in
         let
           capabilities = [
             "CAP_NET_ADMIN"
-          ] ++ lib.optional cfg.settings.dns.enabled "CAP_NET_BIND_SERVICE";
+          ] ++ lib.optional cfg.settings.dns.enable "CAP_NET_BIND_SERVICE";
```

Comment:
It is true that this is a typo, but the typo isn't here it's in line 21. As the settings.* options are RFC 42 style options, we mirror upstream config options. See https://www.freie-netze.org/wg-access-server/2-configuration/#:~:text=dns.enabled

Reply:
Pure technically, the RFC42 compliance is right here.
But this will make a lot of people trip. PR updated.